cmd/snap/cmd_remodel.go

```diff
@@ -36,29 +36,38 @@ revision or a full new model.
 
 In the process it applies any implied changes to the device: new required
 snaps, new kernel or gadget etc.
+
+Snaps and assertions are downloaded from the store unless they can be found
+locally, by looking at files specified by --snap and --assertion options.
```

Comment:
this doesn't make it completely clear that all the snap need to come from the store or locally, one cannot mix?